share/completions/exif.fish

```diff
@@ -0,0 +1,89 @@
+function __get_target_files
+  set param_args (string split ', ' -- (string trim (string replace -fr "=\w+.*" '' -- (exif --help))))
+  set tokens (commandline -po)[2..]
+  set current_token 1
```

Comment:
Also you want these variables to be explicitly local - `set -l current_token`.

Reply:
Ok, I believe this is fixed in 936c9847, I don't see any more variables that leak out after testing.